src/MusicalScore/Graphical/EngravingRules.ts

```diff
@@ -593,10 +597,57 @@ export class EngravingRules {
         chordtexts.setValue(ChordSymbolEnum.suspendedsecond, "sus2");
         chordtexts.setValue(ChordSymbolEnum.suspendedfourth, "sus4");
         chordtexts.setValue(ChordSymbolEnum.power, "5");
+        chordtexts.setValue(ChordSymbolEnum.none, "N.C.");
 
         return chordtexts;
     }
 
+    public addCustomChordKind(
+        altName: string,
+        chordKindText: string,
+        adds: string[],
+        alts: string[],
+        subs: string[],
+    ): void {
+        if (ChordSymbolEnum[chordKindText] !== undefined) {
+            this.CustomChordKinds.push(CustomChordKind.createCustomChordKind(altName, ChordSymbolEnum[chordKindText], adds, alts, subs));
+        }
+    }
+
+    public renameCustomChordKind(
+        altName: string,
+        newAltName: string,
```

Comment:
Please look at our coding conventions (how some of the other code is written) and put the function definition in one line if it has 2(+) arguments or less.